lib/openstudio-standards/standards/necb/NECB2011/necb_2011.rb

```diff
@@ -35,6 +35,12 @@ def convert_arg_to_bool(variable:, default:)
     return true
   end
 
+  def convert_arg_to_string(variable:, default:)
```

Comment:
remove this method.

instead of:
`shw_fuel = convert_arg_to_string(variable: shw_fuel, default: 'NECB_Default')`
use:
`shw_fuel = shw_fuel.nil? ? 'NECB_Default' : shw_fuel.to_s`

or some other sort of if else statement. don't make a new generically named method for something only used once.

Reply:
Revised based on the suggestion.